Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -1235,6 +1236,82 @@ public async Task ExchangeWithIceServer()
             }
         }
 
+        [FactOnlyTurnAvailable(Timeout = Timeout)]
+        public async Task ReconnectToTurn()
+        {
+            const int port = 34567;
```

Comment:
I guess it would be better if we use an assigned port through `new TcpClient(IPAddress.Loopback, 0)` instead of a static number.